plugins/readtimer.koplugin/main.lua

```diff
@@ -80,17 +80,19 @@ function ReadTimer:addToMainMenu(menu_items)
         end,
         sub_item_table = {
             {
-                text = _("Time"),
+                text = _("Point in Time"),
```

Comment:
Could you explain why you want to change this?

(As a generic remark for the future, that should be `Point in time`.)

Reply:
The idea is that in the menu it is called read timer (in my understanding a timer is something similar to a countdown)
In the menu we really have such a timer (counting down) and additionaly an alarm clock.

By explicitly naming both entries as what they do, it is clear to the user what to select.

I think it is also easier for translators by naming the entries explicitly. If we call the first `alarm`, I would assume a German translator to use the German word `Alarm` which is not good at all.

Reply:
BTW: Shouldn't the plugin's menu entry called like something: `Read timer and alarm`? A timer is similar to an alarm but not quite the same?

Reply:
Plugins menu entry: `Timer`
Submenu: `Set time`, `Set interval`, `Stop timer`.

Reply:
@zwim 
> I think it is also easier for translators by naming the entries explicitly. If we call the first alarm, I would assume a German translator to use the German word Alarm which is not good at all.

NB I implemented contexts for a reason.

@hius07 +1 on something like that